python/tvm/relay/op/_transform.py

```diff
@@ -19,16 +19,17 @@
 # pylint: disable=too-many-local-variables, too-many-arguments, no-else-return
 
 from __future__ import absolute_import
+
 import tvm
-from tvm import te
-from tvm.te.hybrid import script
+from tvm import te, topi
 from tvm.runtime import convert
-from tvm import topi
+from tvm.te.hybrid import script
```

Comment:
unused import?

Reply:
It's used elsewhere in this file, for some reason the autoformater moved it around in the input list